DuckDuckGo/TabSwitcherViewController.swift

```diff
@@ -518,9 +518,12 @@ extension TabSwitcherViewController: TabObserver {
             return
         }
 
-        if let index = tabsModel.indexOf(tab: tab), index < collectionView.numberOfItems(inSection: 0) {
-            collectionView.reconfigureItems(at: [IndexPath(row: index, section: 0)])
-        }
+        collectionView.performBatchUpdates({}, completion: { [weak self] completed in
```

Comment:
Interesting, so in this case we aren't actually making any batch changes, but rather just waiting for the completion? Not sure I totally follow it, but functionally it works great for me locally.

Reply:
yeah my thinking is if we're trying to reconfigure while some other batch is in progress then it's safer to wait. tbh it's just a guess so we'll see how it goes for a week before it gets released.  Thanks.